### DISCUSSION TITLE
Welcome to esbuild-jest Discussions!

### DISCUSSION THREAD

**Discussion by aelbore** [aelbore/esbuild-jest#62] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JoaquinFernandez** [comment]
Hi! I see there are some PRs pending and that the transformer may not be maintained anymore?

Maybe it would make sense to add someone else as maintainer for this?

**Comment by samhwang** [comment]
Hey Jay, thanks for the effort to create this plugin for Jest, but it looks like it's being abandoned :( Is there any plans for you to continue working on this project?
